src/table/Table.tsx

```diff
@@ -193,10 +201,31 @@ export const Table = <T extends object>(props: TableProps<T>) => {
                       width: column.width,
                       maxWidth: column.maxWidth,
                     },
+                    ...(column.getSortByToggleProps
+                      ? column.getSortByToggleProps()
+                      : {}),
                   })}
                   key={j}
                 >
-                  {column.render('Header')}
+                  <div tw="flex items-center ">
+                    {column.render('Header')}
+                    {column.canSort &&
+                      (() => {
+                        return (
+                          <div>
+                            {column.isSorted ? (
+                              column.isSortedDesc ? (
+                                <MdArrowDownward tw="ml-1" />
+                              ) : (
+                                <MdArrowUpward tw="ml-1" />
+                              )
+                            ) : (
+                              ''
```

Comment:
Nits: 動かないわけではないと思うのですが、
```suggestion
                              <></>
```
の方が良いのかなと思いました

Reply:
たしかにそうですね！ <></>だとたしか空のDivができちゃうので、Nullにしておきました！
174434b8acb7431cdba9fb5941ccc4dcd5a93e6c